redismodule.h

```diff
@@ -0,0 +1,405 @@
+#ifndef REDISMODULE_H
```

Comment:
Is it necessary for this file to be in the top level as opposed to a subdirectory? If so, why? Also: I'm assuming this is boilerplate for any redis module and doesn't need review, please let me know if that's not the case.

Reply:
I can't really explain this. I didn't write the redismodule.h header file (it's something that you have to add to every directory that creates a redis module). This header file is actually created in the RedisModulesSDK directory which is put out by RedisLabs (https://github.com/RedisLabs/RedisModulesSDK/tree/3e4daab6dcbc881d8c17406235d156de55be4fc7). Also to possibly address the idea of adding a submodule instead, I didn't think of it while writing this code and am planning to add it in a future commit/PR (along with other optimizations for this module, I just wanted to put out a working product before anything else).

Reply:
Gotcha, OK, I think we can leave it here for now.